### ISSUE TITLE
Wrong namespace on index.d.ts export

### ISSUE THREAD

**Issue by suncodefactory** [twbs/bootstrap#33208]
I am trying to use "bootstrap" namespace like this bootstrap.Modal.getInstance(modalElement)
But my typescript files does not compile because "bootstrap" doe snot exist.
Further checking I found that the index.d.ts file in the @types/bootstrap folder in my node_modules
exports the namespace like this: export as namespace Bootstrap; which is incorrect.
When I change to  Bootstrap.Modal.getInstance(modalElement) it compiles but fails at runtime.

So basically the export Bootstrap vs bootstrap (with lower case) is what is breaking my typescript.

I was to bypass the issue by manually editing the index.d.ts file to use lower case export as namespace bootstrap

I am using v5 beta2 if that makes any difference.

**Comment by XhmikosR** [comment]
We don't provide TypeScript definitions ourselves. You should report such issues to the proper repo.
